CMakeLists.txt

```diff
@@ -713,7 +713,6 @@ if (USE_TORCH)
     ${PYTORCH_PATCHES_PATH}/pytorch_16_new_logger.patch
     ${PYTORCH_PATCHES_PATH}/pytorch_16_use_new_logger.patch
     ${PYTORCH_PATCHES_PATH}/pytorch_14_namespace.patch
-    ${PYTORCH_PATCHES_PATH}/pytorch_16_fatbin.patch
```

Comment:
Maybe delete the file too ?

Reply:
not sure about git behavior, if this file is deleted, can we recover it from git history ?